internal/provider/private_endpoint_services_resource.go

```diff
@@ -76,8 +76,22 @@ var endpointServicesSchema = schema.Schema{
 						Computed:    true,
 						Description: "Operation status of the service.",
 					},
+					"name": schema.StringAttribute{
+						Computed:    true,
+						Description: "Name of the endpoint service.",
+					},
+					"endpoint_service_id": schema.StringAttribute{
+						Computed:    true,
+						Description: "Server side ID of the private endpoint connection.",
+					},
+					"availability_zone_ids": schema.ListAttribute{
+						Computed:            true,
+						ElementType:         types.StringType,
+						MarkdownDescription: "AZ IDs users should create their VPCs in to minimize their cost.",
+					},
 					"aws": schema.SingleNestedAttribute{
-						Computed: true,
+						DeprecationMessage: "these fields have are now contained in the outer fields: name, endpoint_service_id, availability_zone_ids",
```

Comment:
nit: grammar 

maybe something like: 
`nested aws fields have been moved one level up.  These fields will be removed in a future version`

Reply:
done.